doc/devel/coding_guide.rst

```diff
@@ -148,12 +148,25 @@ We do a backport from master to v2.2.x assuming:
 
 The ``TARGET_SHA`` is the hash of the merge commit you would like to
 backport.  This can be read off of the github PR page (in the UI with
-the merge notification) or through the git CLI tools.::
+the merge notification) or through the git CLI tools.
 
-  git fetch matplotlib
-  git checkout v2.2.x
-  git merge --ff-only matplotlib/v2.2.x
+Assuming that you already have a local branch ``v2.2.x`` (if not, then
+``git checkout -b v2.2.x``), and that your remote pointing to
+``https://github.com/matplotlib/matplotlib`` is called ``upstream``::
+
+  git fetch upstream
+  git checkout v2.2.x  # or include -b if you don't already have this.
+  git reset --hard upstream/v2.2.x
   git cherry-pick -m 1 TARGET_SHA
   # resolve conflicts and commit if required
 
-Use your discretion to push directly to upstream or to open a PR.
+The conflicts will have to be searched for by hand, (search on ``>>>>>``)
+and if a file is conflicted, you will have to re-add to the branch and
+make a commit::
+
+  git add lib/matplotlib/conflicted_file.py
+  git add lib/matplotlib/conflicted_file2.py
+  git commit -a -m "MRG: backport of SHA"
```

Comment:
Don't think you should add `-a`. You've already added the files manually.

Reply:
1. This is in conflict with the text above: : "When doing backports please copy the form used by meeseekdev, Backport PR #XXXX: TITLE OF PR".
2. Do we use prefixes like MRG for commit messages? If so, where's that documented?

Reply:
Thanks!  Both comments fixed....

Reply:
... good thing I self merged my mis-titled backport PR :wink: